### DISCUSSION TITLE
アルバムを再生で1曲しか再生されず、全曲再生されない。アルバムの全曲を自動で再生させる設定を教えて下さい。

### DISCUSSION THREAD

**Discussion by eco-pimento** [TT-ReBORN/Georgia-ReBORN#56] [Q&A] [answered]
Playing an album only plays one song, not all songs. Please tell me the setting to play all songs in the album automatically.
Why can't I play all songs in the album with the default settings? Is it an error? Or is it such a setting?

**Comment by silverwings15** [comment, marked as answer]
seems like you're only adding one track to the playlist each time. Click twice on "Library" and see what your playlist looks like

**Reply by TT-ReBORN** [reply]
Hi @eco-pimento,

silverwings15 is right, other users have already asked about this multiple times. 
I have added a new FAQ section here on the forum: [FAQ](https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/58)

Please visit it, you will find detailed instructions how it works.

-TT

**Reply by eco-pimento** [reply]
Thank you, TT-ReBORN, for the appropriate answer. First of all, it seems like double-clicking an album to recognize the track. I didn't know what to do without this work in older versions of foober. It's working fine.

**Reply by eco-pimento** [reply]
Is it Javascript? I can't understand the contents of the email due to my lack of knowledge. I'm sorry for your email.
Don't be offended. Thank you so much.

**Comment by kbuffington** [comment]
There's also a possibility that "Playback >> Stop After Current" is checked.

**Reply by silverwings15** [reply]
Stop After Current doesn't persist after it takes effect once, at least not ime, unless OP ran into the issue just one time and already decided to make a thread about it . on that note, would be pretty sweet if Georgia/ReBORN have some sort of built-in visual indicator for Stop After Current being active

**Reply by kbuffington** [reply]
Ahh, you're right. Completely forgot about that 😀

**Reply by eco-pimento** [reply]
I learned from TT-ReBORN. A common problem was that you first had to double-click on an album to recognize the track. I didn't know what to do without this work in older versions of foober. It's working fine.

**Comment by eco-pimento** [comment]
Is it Javascript? I can't understand the contents of the email due to my lack of knowledge. I'm sorry for your email.

Don't be offended. Thank you very much.

**Reply by TT-ReBORN** [reply]
Yes,
it was a Javascript related question about some code and was off topic. This message was not for you but for a kbuffington, 
problem already solved thanks to him =)

Sorry if I have confused you...

-TT
